features/volgindicaties.feature

```diff
@@ -0,0 +1,154 @@
+# language: nl
+
+Functionaliteit: Als gebruiker wil ik wijzigingen kunnen opvragen van zowel binnengemeentelijke als buitengemeentelijke personen
+    Om wijzigingen te ontvangen van buitengemeentelijke personen, moet een volgindicatie worden gezet in GBA-V middels een Ap01 bericht.
+    Een volgindicatie in GBA-V wordt beëindigd met een Av01 bericht.
+    
+    Hierbij gelden 3 hoofdregels:
+    1. Wanneer er ten minste één gebruiker een actieve volgindicatie heeft op een buitengemeentelijke persoon, moet er ook een actieve volgindicatie zijn voor deze persoon bij het GBA-V.
+    2. Wanneer geen enkele gebruiker een actieve volgindicatie op een buitengemeentelijke persoon, mag er geen actieve volgindicatie zijn voor deze persoon bij het GBA-V.
+    3. Voor een binnengemeentelijke persoon mag er geen actieve volgindicatie zijn in het GBA-V.
+
+   Volgindicaties in GBA-V kunnen worden gewijzigd n.a.v.:
+      - Een PUT /volgindicaties request
+      - Het bereiken van een datum
+      - Een binnengemeentelijke wijziging (de update applicatie ontvangt van een StUF npsLk01 kennisgeving uit het burgerzakensysteem)
+        - persoon is nu ingeschreven in deze gemeenten en applicatie ontvangt StUF mutatiesoort='V': uitverhuizing
+        - persoon is nu ingeschreven in andere gemeenten en applicatie ontvangt StUF mutatiesoort≠'V' (normaliter 'T'): inverhuizing
+ 
+
+   Scenario: Een gebruiker zet als eerste een volgindicatie op een binnengemeentelijk persoon
+      Gegeven de persoon met burgerservicenummer "999995066" is ingeschreven in deze gemeente
+      En er is geen enkele volgindicatie gezet voor de persoon met burgerservicenummer "999995066"
+      Als gebruiker "A" op 5 maart 2021 een volgindicatie zet op de persoon met burgerservicenummer "999995066" en geen einddatum
+      Dan stuurt de applicatie GEEN Ap01 bericht met burgerservicenummer "999995066" naar het GBA-V 
```

Comment:
Wordt met applicatie de Update API bedoeld?

Reply:
ja, voor mij is de Update API de interface naar clients, en zit er daarachter een applicatie die StUF berichten ontvangt en communicatie met VOA doet. Zou jij dat deel ook "Update API" noemen? Of "Update applicatie" of iets dergelijks?

Reply:
ik wijzig het nu overal naar "update applicatie"